spec/fixtures/invalid_lint_template.json

```diff
@@ -0,0 +1,21 @@
+{
+    "AWSTemplateFormatVersion" : "2010-09-09",
+    "Resources" : {
+       "myVPC" : {
+          "Type" : "AWS::EC2::VPC",
+          "Properties" : {
+            "CidrBlock" : 1,
+            "EnableDnsSupport" : [],
+            "EnableDnsHostnames" : [],
+            "InstanceTenancy" : {},
+            "Tags" : [ {"Key" : "foo", "Value" : "bar"} ]
+          }
+       }
+    },
+    "Outputs" : {
+      "VpcId": {
+        "Value": {"Ref": "myVPC"}
+      }
+    }
+  }    
+  
```

Comment:
It's not quite clear if this template is not valid because of the missing `}` or is there supposed to be a mistake in it.

If it's not valid json, can we strip it to just `{`? Otherwise, can we fix that and put what's actually wrong in the filename?

Reply:
It is valid JSON just the whitespace was skewed. Fixed!